service/tasks/monitoring.py

```diff
@@ -548,20 +529,6 @@ def monitor_instances():
         monitor_instances_for.apply_async(args=[p.id])
 
 
-@task(name="enforce_allocation_overage")
```

Comment:
deleted enforce_allocation_overage
1) Open stack enfoces the allocation - our UI should just report it.
2) Causes 2 undefined variables:
    a) 'entity_id=source.source_id' should be 'entity_id=allocation_source.source_id'
    b) Don't know what new_payload should be - Though it is probably the following:
        core/hooks/allocation_source.py (53)
            new_payload = {
                "allocation_source_id": source.source_id,
                "actual_value": current_percentage
            }
3) Won't ever do anything useful for us.
4) wasn't being called anyways (only called from core/hooks/allocation_source.py).